prac_08/box_layout_demo.py

```diff
@@ -10,6 +10,12 @@ def build(self):
 
     def handle_greet(self):
         print('greet')
+        self.root.ids.output_label.text = "Hello"
+        self.root.ids.output_label.text = f"Hello {self.root.ids.input_name.text}"
 
+    def handle_clear(self):
+        self.root.ids.output_label.text = ""
+        self.root.ids.input_name.text = ""
 
-BoxLayoutDemo().run()
+
+BoxLayoutDemo().run()
```

Comment:
this line was apart of the code or? sorry to ask i just got a little confused for a second haha